### PR TITLE
Minor Tweaks and Fixes

### PR DESCRIPTION
**Pull request by Light-Weave**
## Description
See below.

## Motivation and Context
Please. See below. There's so many. @ me in discord for individual explanations.

## How Has This Been Tested?
All thoroughly tested.

## Changelog (necessary)
:cl:
tweak: Adds ear protection to NCR, BOS, Vault-Sec and Enclave headsets. 
tweak: Power armor helmets and Veteran Ranger helmets now have built-in low-light vision. Veteran Ranger helmets can be alt-clicked to provide a snazzy red overlay.
tweak: Powerfist fits on the belt slot.
tweak: Vexillarius now spawns with a Winchester trench shotgun.
tweak: Patrol Rangers now spawn with binoculars like Explorers.
tweak: The box of 40mm grenades in the NCR armory has been replaced with four flashbangs.
tweak: Equalizes amount of 9mm and 5.56 magazines in NCR armory.
tweak: The boxes of single shotguns in the Legion armory have been replaced with glaives and shields.
tweak: The power plant now has a random gun spawner.
spellcheck: Lethal shotgun box renamed to "box of buckshot shotgun shots".
fix: Bumper sword is now named correctly.
fix: NVGs now work properly with increased view ranges.
fix: Preacher now spawns with ammo.
fix: Recon Ranger now spawns with a knife.
fix: The (unusable) legcuffs in the NCR prison have been replaced with handcuffs.
balance: Wooden bucklers have been rebalanced to provide 30/20/10 melee/bullet/laser armor with a 25% block chance.
add: Adds an ammo bench to the gunstore backroom, used for crafting ammo for weapons. The Den lathe is cheaper, though.
add: The NCR base now has a small firing range by the gate.
add: The Legion base now has two spades for digging sand and making fortifications.
add: The Centurion's office has a wrench for use on his powerfist.
add: Two destroyed Ranger outposts can be found in Legion territory, along with the creatures that overran them.
del: The Klamat autolathe has been removed.
/:cl:

### CODE REVIEW

**Comment by Light-Weave** [comment]
Merge conflicts from the routine maintenance and Legion rank structure PR. Both PRs have been integrated into this one- it's safe to merge even if it conflicts (at least, until someone else touches those files before this).

**Comment by UndeadNappist** [comment]
The removal of the autolathe and replacement with the constructionlathes in klamat, for the NCR, and even in the gun store were done for specific and explicit purposes.

The discussion essentially boiled down to "I want people to stop resorting to guns as their first choice" and "I want the threat of running out of ammo for each faction to be an actual problem."

Adding the gun reloading bench to the game seems to be a step backwards, especially considering the current problems with not only chemistry but also being able to deconstruct certain things for metal when you really, really shouldn't be able to.

After all the constructionlathe explicitly had it's ability to make ammo and replaced the autolathes for these explicit reasons.

**Comment by Light-Weave** [comment]
Ammo bench has been moved to the gunstore, and I'd swap the legcuffs for regcuffs but I *really* don't wanna touch the map merger again... tomorrow!

**Comment by UndeadNappist** [comment]
No problem. You've done a lot of great work already and I'm pretty happy with it.

**Comment by Light-Weave** [comment]
honestly I'd prefer a merge as soon as the 24h hold is up, just because the map merger is broken and this means nobody else can touch the map in the meantime

(and also this touches a ton of files)

**Comment by Light-Weave** [comment]
nvm map merger was fine i'm just an idiot

File: code/modules/power/lighting.dm [hunk 1]
```diff
@@ -363,7 +363,7 @@
 			addStaticPower(static_power_used, STATIC_LIGHT)
 		else
 			removeStaticPower(static_power_used, STATIC_LIGHT)
-
+	night_update()
```
Comment (LetterN) [inline code/modules/power/lighting.dm]: delete dis

Comment (Light-Weave) [reply code/modules/power/lighting.dm]: 👍

**Comment by Light-Weave** [comment]
Nightvision from Veteran Ranger and Power Armor helmets replaced with limited low-light vision. Actual NVGs are much more powerful.

**Comment by UndeadNappist** [comment]
Trashbag a bad crafting recipe, it's the best container in the game. Shouldn't be craftable, only loot.

**Comment by Light-Weave** [comment]
Yeah, that one's been removed, I just haven't pushed the commit yet. Also giving LLVGs a red overlay per request (altclick the helmet to toggle).

File: code/modules/crafting/recipes.dm [hunk 2]
```diff
@@ -334,6 +458,14 @@
...
 	subcategory = CAT_WEAPON
 
+/datum/crafting_recipe/trashbag
+	name = "Trash Bag"
+	result = /obj/item/storage/bag/trash
+	time = 15
+	reqs = list(/obj/item/stack/sheet/plastic = 3)
+	tools = list(/obj/item/kitchen/rollingpin)
+	category = CAT_MISC
+
```
Comment (UndeadNappist) [inline code/modules/crafting/recipes.dm]: remuv for reasons stated ; leaving this here so it doesn't get accidental merged

File: code/modules/research/designs/autolathe_designs.dm [hunk 3]
```diff
@@ -420,6 +420,14 @@
...
 	category = list("initial", "Medical")
 
+/datum/design/solids
+	name = "Solid Beaker"
+	id = "solids"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 500)
+	build_path = /obj/item/reagent_containers/glass/beaker/solids
+	category = list("initial", "Medical")
+
```
Comment (UndeadNappist) [inline code/modules/research/designs/autolathe_designs.dm]: Moving solid beakers which are only explicitly for making grenades and are objectively worse than normal beakers.
No.
If you want to do something like this, just add glass bottles (30u) as an available option for grenades.

File: code/modules/crafting/recipes.dm (hunk 2 above)
Comment (Light-Weave) [reply code/modules/crafting/recipes.dm]: 👍

File: code/modules/research/designs/autolathe_designs.dm (hunk 3 above)
Comment (Light-Weave) [reply code/modules/research/designs/autolathe_designs.dm]: :+1: done

**Comment by Light-Weave** [comment]
you haven't lived until you've made 13 separate commits to one branch
anyway it's ready to merge now unless i add something else in the next 2 hours

**Pull request merged**